### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by badersur**

### CODE REVIEW

File: docs/csharp/tutorials/index.md
Comment (mairaw) [inline docs/csharp/tutorials/index.md]: Wondering if it would be even better to say:"

The following tutorials enable you to build C# programs using [CoreCLR](https://github.com/dotnet/coreclr):

File: docs/csharp/tutorials/index.md
Comment (mairaw) [inline docs/csharp/tutorials/index.md]: Since we're already saying each of these are tutorials in the introduction, maybe we could cut the first part of the sentence and say:
[Console Application](console-teleprompter.md): demonstrates Console I/O, ....
It sounds less repetitive to me.

**Comment by mairaw** [comment]
Thank you @badersur. The changes look good but I had a few comments on how I think we could make the topic even better. I'll close and reopen this PR since one of the CI builds had failed.

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tutorials/index?branch=pr-en-us-1049](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tutorials/index?branch=pr-en-us-1049)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C20%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609201856587231-1049%5Cworkflow_report.html).

File: docs/csharp/tutorials/index.md
Comment (badersur) [inline docs/csharp/tutorials/index.md]: And it seems better to me. will change it.

File: docs/csharp/tutorials/index.md
Comment (badersur) [inline docs/csharp/tutorials/index.md]: I thought about changing it but changed my mind. Will change it.

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tutorials/index?branch=pr-en-us-1049](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tutorials/index?branch=pr-en-us-1049)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C20%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609201908251203-1049%5Cworkflow_report.html).

**Review by mairaw — APPROVED** [review]
LGTM @badersur! Thanks for making the changes. Let's wait until the CI builds are done to merge this.

**Comment by badersur** [comment]
You are welcome @mairaw :)

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tutorials/index?branch=pr-en-us-1049](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tutorials/index?branch=pr-en-us-1049)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C20%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609201924226552-1049%5Cworkflow_report.html).

**Pull request merged**
